docker/entrypoint_alpine.sh

```diff
@@ -49,5 +49,7 @@ php artisan migrate --force
 php artisan config:clear
 php artisan config:cache
 
+chown -R apache:root /var/www/html/storage/logs/laravel.log
```

Comment:
I assume this must just be another permissions bug we're fixing - but this wasn't here before so I just want to make sure...

Reply:
I added this line because after a forced reboot of the container, the ownership of this file was transferred to the root user, preventing a new initialization. In a normal startup this line is not necessary